index.html

```diff
@@ -61,6 +61,172 @@ <h2 class="body-text-header">I'm a Software Developer</h2>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      </section>
+      <section class="about">
+        <h2 class="about-heading">About me</h2>
+        <p class="about-disc">Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experienc! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <button class="about-btn">Get My Resume</button>
```

Comment:
- [ ] As indicated in other sections above, please endeavor to set the button type to `button` or `submit` for proper semantic scores and accessibility boosting :+1: